### PR TITLE
cluster-operator: prepare master branch

### PR DESCRIPTION
**Pull request by kopiczko**
Towards https://github.com/giantswarm/giantswarm/issues/7722.

⚠️ https://github.com/giantswarm/cluster-operator/pull/799 should be merged first.

- Drop legacy controller. key packages in {aws,azure,kvm}/v22/key still
  there. They are used by resources configured with provider. This will
  be flattened in follow up PRs.
- Drop clusterapi 0.21.0 (v21).

Code changes:
- Two imports changed in `service/service.go` and `service/collector/cluster.go`.
- Whitespace changes after funning `go fmt ./...`.

Other than that there are no added lines in this PR. Only removals.

### CODE REVIEW

File: service/service.go [hunk 1]
```diff
@@ -28,12 +28,9 @@ import (
 	"github.com/giantswarm/cluster-operator/flag"
 	"github.com/giantswarm/cluster-operator/pkg/cluster"
 	"github.com/giantswarm/cluster-operator/pkg/label"
-	"github.com/giantswarm/cluster-operator/pkg/v19/key"
+	"github.com/giantswarm/cluster-operator/pkg/v22/key"
```
Comment (kopiczko) [inline service/service.go]: This thing changed.

File: service/collector/cluster.go [hunk 2]
```diff
@@ -9,7 +9,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset"
 
-	"github.com/giantswarm/cluster-operator/service/controller/clusterapi/v21/key"
+	"github.com/giantswarm/cluster-operator/service/controller/clusterapi/v22/key"
```
Comment (kopiczko) [inline service/collector/cluster.go]: This thing changed.

**Comment by xh3b4sd** [comment]
My browser tab just crashed trying to scan the diff. Why should we drop the clusterapi in master? This is the new thing?

**Comment by kopiczko** [comment]
> My browser tab just crashed trying to scan the diff. Why should we drop the clusterapi in master? This is the new thing?

It should be `Drop clusterapi 0.21.0 (v21).`.  Updated description.

**Pull request merged**